### ISSUE TITLE
tkctl: No TiDB Controller Manager found, please install one first

### ISSUE THREAD

**Issue by weekface** [pingcap/tidb-operator#1028]
The TiDB Operator installed with releaseName "tidb-operator", `tkctl verison` return error:

```
# tkctl version
Client Version: v1.1.0-alpha.1.67+5f1df4c30ba561
No TiDB Controller Manager found, please install one first
No TiDB Scheduler found, please install one first
```

Because the `labelSelector` of `ListOptions` was written with `app.kubernetes.io/instance: operator`:

https://github.com/pingcap/tidb-operator/blob/b6a87973feb6848db075b1baf8bdde322c5f443c/pkg/tkctl/cmd/version/version.go#L83

https://github.com/pingcap/tidb-operator/blob/b6a87973feb6848db075b1baf8bdde322c5f443c/pkg/tkctl/cmd/version/version.go#L91

**Comment by weekface** [comment]
https://download.pingcap.org/tkctl-linux-amd64-latest.tgz

This link is not the latest binary.

**Comment by tennix** [comment]
You must have downloaded the cached version. I've contacted with the EE team to disable CDN cache for this file.

**Comment by aylei** [comment]
@weekface The latest code specify the instance value should be equal to `operator`, so the binary is actually the latest one.

However, this assumption is not always true, we should refine the logic of `kubectl version`

**Comment by aylei** [comment]
closed via #1065
